proxy-node-gateway/src/main/java/uk/gov/ida/notification/resources/AssetsResource.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ida.notification.resources;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+@Path("/")
+public class AssetsResource {
```

Comment:
This would have made a superb #good-first-issue card for a dev new to DropWizard ;)

Reply:
Actually, it would probably be more elegant to handle those requests with a filter rather than have a whole new resource so feel free to take it on as a "learning Dropwizard" side project and make it better :)